lib/infinum_azure/version.rb

```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
 
 module InfinumAzure
-  VERSION = '3.0.0'
+  VERSION = '4.0.0'
```

Comment:
this is a new feature I'd say, not a breaking change, so let's release 3.1.0. (don't forget to run bundle install after changing this file)